src/features/store.ts

```diff
@@ -26,6 +26,7 @@ interface WarpContext {
   multiProvider: MultiProtocolProvider;
   warpCore: WarpCore;
   tokensBySymbolChainMap: Record<string, TokenChainMap>;
+  routerAddresses: Set<string>;
```

Comment:
Wouldn't it be better to have a mapping/record where the key is the chain name and the value is a set with the warp route token addresses deployed on that chain instead of a single set?